content/src/components.ts

```diff
@@ -40,8 +41,12 @@ export async function initComponentsWithEnv(env: Environment): Promise<AppCompon
   const repository = await RepositoryFactory.create({ env, metrics })
   const logs = createLogComponent()
   const fetcher = createFetchComponent()
+  const contentStorageFolder = path.join(env.getConfig(EnvironmentConfig.STORAGE_ROOT_FOLDER), 'contents', '_tmp')
+  const tmpDownloadFolder = path.join(contentStorageFolder, '_tmp')
```

Comment:
I think tmpDownloadFolder is `xxx/contents/_tmp/_tmp`

Reply:
thxs, fixed.